public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
```

Comment:
Just for info ElasticSearch can return highlighting fragments as part of a query response in a way which is configurable.